test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -1005,23 +930,23 @@ def __copy_dir_with_core(self, host, src_path, dest_path, core_dir,
         When a core file is found, capture the stack trace from it
 
         :param host: name of the host where the source is located
-        :type String
+        :type str
         :param src_path: path of the source directory
-        :type String
+        :type str
         :param dest_path: path of the destination directory
-        :type String
+        :type str
         :param core_dir: path to the directory to store core files' trace
-        :type String
+        :type str
         :param except_list: list  of files/directories (basenames) to exclude
-        :type List
+        :type list
         :param only_core: Copy over only core files?
-        :type Boolean
+        :type bool
         """
         dir_list = None
         try:
             dir_list = self.du.listdir(host, src_path, fullpath=False,
                                        sudo=self.sudo)
-        except:
+        except Exception:
```

Comment:
@bhroam Like @agrawalravi90, I also think that just 'except' is bad practice as it will catch all 'BaseException' and as per BaseException's document and PEP 8 (see point 6, 9, 10 and 11 at [here](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)) is something that user should not catch unless it is very much require. In other word user should only catch 'non-system-exiting' exceptions only not system-exiting exceptions like SystemExit, GeneratorExit etc... and BaseException is base class for system-exiting exceptions.